k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageContainerView.java

```diff
@@ -380,7 +379,7 @@ private void setLoadPictures(boolean enable) {
 
     public void showPictures() {
         setLoadPictures(true);
-        loadBodyFromText(mText);
+        displayHtmlContentWithInlineAttachments(mText, null);
```

Comment:
Passing `null` as second argument feels wrong. We're setting the same text, why not use the same `AttachmentResolver`?